File: samples/cs/01table/table.cs
using System;

namespace Table {
    class Exec {
        const int LIMIT = 5;
        static void Main(string[] args) {
            Console.Write("   ");
            for (int i = 1; i <= LIMIT; i++) {
                Console.Write(i.ToString().PadRight(3));
            }
            Console.WriteLine();
            
            for (int a = 1; a <= LIMIT; a++) {
                Console.Write(a.ToString().PadRight(3));
                for (int b = 1; b <= LIMIT; b++) {
                    Console.Write((a*b).ToString().PadRight(3));
                }
                Console.WriteLine();
            }
        }
    }
}

File: samples/java/01table/table.java
public class table {
    public static final int LIMIT = 5;
    public static void main(String[] args) {
        System.out.print("   ");
        for (int i = 1; i <= LIMIT; i++) {
            System.out.print(String.format("%-3d", i));
        }
        System.out.println();
        
        for (int a = 1; a <= LIMIT; a++) {
            System.out.print(String.format("%-3d", a));
            for (int b = 1; b <= LIMIT; b++) {
                System.out.print(String.format("%-3d", a*b));
            }
            System.out.println();
        }
    }
}
